data/moves.js

```diff
@@ -14119,17 +14119,13 @@ exports.BattleMovedex = {
 		priority: 0,
 		flags: {protect: 1, mirror: 1, nonsky: 1},
 		isUnreleased: true,
-		onModifyMove: function (move) {
-			if (move.type === 'Ground') {
-				move.affectedByImmunities = false;
-			}
-		},
 		onTryHit: function (target) {
 			// only the attack that grounds the target ignores effectiveness
 			if (target.negateImmunity['Ground']) return;
 			target.negateImmunity['Ground'] = 'IgnoreEffectiveness';
 		},
 		volatileStatus: 'smackdown',
+		ignoreImmunity: {'Ground': true},
```

Comment:
This change in particular seems like a good idea as it both makes the information about the move easier to access and also easier to read and understand. While Thousand Arrows might currently be a special case move, I would not be surprised if more moves that could use this notation will appear in the future.